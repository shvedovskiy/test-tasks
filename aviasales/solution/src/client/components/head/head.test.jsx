/* eslint-disable no-undef */
import React from 'react';

import Head from './head';


describe('<Head />', () => {
  const head = shallow(<Head />);

  it('renders properly', () => {
    expect(head).toMatchSnapshot();
  });
});
