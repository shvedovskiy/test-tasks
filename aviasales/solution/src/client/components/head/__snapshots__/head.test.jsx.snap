// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Head /> renders properly 1`] = `
<styled__Header>
  <styled__Logo
    aria-label="Logo picture"
    href="/"
    title="Homepage"
  />
</styled__Header>
`;
